tests/test/faaslet/test_io.cpp

```diff
@@ -8,28 +8,35 @@
 using namespace faaslet;
 
 namespace tests {
-TEST_CASE("Test fixed input with colon", "[faaslet]")
+TEST_CASE("Test fixed input with colon", "[faaslet][wamr]")
 {
     cleanSystem();
     faabric::Message call =
       faabric::util::messageFactory("demo", "check_input");
     call.set_inputdata("http://www.foobar.com");
 
-    execFunction(call);
+    SECTION("WAVM") { execFunction(call); }
+
+    SECTION("WAMR") { execWamrFunction(call); }
 }
 
-TEST_CASE("Test execution of echo function", "[faaslet]")
+TEST_CASE("Test execution of echo function", "[faaslet][wamr]")
 {
     cleanSystem();
     faabric::Message call = faabric::util::messageFactory("demo", "echo");
     std::string inputData = "http://www.testinput/foo.com";
     call.set_inputdata(inputData.c_str());
 
+    conf::FaasmConfig& conf = conf::getFaasmConfig();
+    SECTION("WAVM") { conf.wasmVm = "wavm"; }
+
+    SECTION("WAMR") { conf.wasmVm = "wamr"; }
```

Comment:
These tests are now changing the config and not resetting it at the end (leaving the `wasmVm` set to `wamr` which may break other tests). 

They should be `TEST_CASE_METHOD`s using a custom fixture inheriting from `FaasmConfTestFixture`  and `FunctionExecTestFixture` (so we can potentially remove the `cleanSystem` too).